src/GraphQL/Utilities/NameValidator.cs

```diff
@@ -8,9 +7,6 @@ namespace GraphQL.Utilities
     /// </summary>
     public static class NameValidator
     {
-        private const string RESERVED_PREFIX = "__";
-        private const string NAME_RX = @"^[_A-Za-z][_0-9A-Za-z]*$";
```

Comment:
I can only guess why there was an underscore in the regular expression, if it is already checked separately at the beginning of the method.

Reply:
Ah, `__` != `_`